### PR TITLE
Added e2e testing exercise (2c)

### LINKED ISSUE #172: Add E2E Testing for the feature in exercise 3

### PR DESCRIPTION
**Pull request by Tompage1994**
Closes #172

### CODE REVIEW

File: exercises/3-revenge-of-the-automated-testing/README.md [hunk 1]
```diff
@@ -566,7 +566,40 @@ git push --all
 
 #### 2c - Create todolist e2e tests
 
-3. TODO !!
+> Using [Nightwatch.js](http://nightwatchjs.org/) We will write a reasonably simple e2e test to test the functionality of the feature we just implimented.
```
Comment (acammies) [inline exercises/3-revenge-of-the-automated-testing/README.md]: *implemented

**Comment by springdo** [comment]
@acammies  - when you reveiwed this; did you test it?

**Comment by Tompage1994** [comment]
@springdo @acammies of course he didn't... @acammies could you do so please 😀

**Comment by acammies** [comment]
Tested and it works fine for me! Put a few comments on some small bits, mainly having the tests as bash comments as opposed to screenshots. Other than that :+1:

**Pull request merged**